### PR TITLE
Update of the logs functionality.

### PR DESCRIPTION
**Pull request by OlegGordiichuk**
- Add new logs events
- Code cleaning 
- Remove of NSLog and print events from code

Fix #706

### CODE REVIEW

File: Trust/Core/Analytics/AnalyticsEvent.swift [hunk 1]
```diff
@@ -70,6 +107,28 @@ enum AnalyticsEvent {
             return ["error": error.prettyError]
         case .backedUpWallet:
             return [:]
+        case .failedTokenFetchRequest(let error):
```
Comment (vikmeup) [inline Trust/Core/Analytics/AnalyticsEvent.swift]: Can you combine all these together into one case? With error

File: Trust/Core/Analytics/AnalyticsEvent.swift [hunk 2]
```diff
@@ -70,6 +107,28 @@ enum AnalyticsEvent {
...
+            return ["error": error.prettyError]
+        case .failedAssetsFetchRequest(let error):
+            return ["error": error.prettyError]
+        case .faileBalanceFetchRequest(let error):
+            return ["error": error.prettyError]
+        case .failedToExcludeFromBackup(let error):
+            return ["error": error.prettyError]
+        case .scriptRunning(let script):
+            return ["script": script]
+        case .failedToLoadTrustPageProvider:
```
Comment (vikmeup) [inline Trust/Core/Analytics/AnalyticsEvent.swift]: Same for these, that has empty return

File: Trust/Browser/Factory/WKWebViewConfiguration.swift [hunk 3]
```diff
@@ -19,10 +19,10 @@ extension WKWebViewConfiguration {
             do {
                 js += try String(contentsOfFile: filepath)
             } catch {
-                NSLog("Failed to load Trust in page provider")
+                Analytics.track(.failedToLoadTrustPageProvider)
             }
         } else {
-            NSLog("Trust in page provider not found in bundle")
+            Analytics.track(.failedToLoadTrustPageProviderFromBundle)
```
Comment (vikmeup) [inline Trust/Browser/Factory/WKWebViewConfiguration.swift]: Can we remove these two? I don’t think there is a neeed for it

File: Trust/Browser/Types/DappAction.swift [hunk 4]
```diff
@@ -16,8 +16,8 @@ enum DappAction {
 
 extension DappAction {
     static func fromCommand(_ command: DappCommand, requester: DAppRequester) -> DappAction {
-        NSLog("command.name \(command.name)")
-        NSLog("command.object \(command.object)")
+        Analytics.track(.command(command.name.rawValue))
+        Analytics.track(.command(command.object.description))
```
Comment (vikmeup) [inline Trust/Browser/Types/DappAction.swift]: Can you remove this login? We don’t need it in general

File: Trust/Browser/ViewControllers/BrowserViewController.swift [hunk 5]
```diff
@@ -146,7 +146,7 @@ class BrowserViewController: UIViewController {
                 return "executeCallback(\(callbackID), \"\(error)\", null)"
             }
         }()
-        NSLog("script \(script)")
+        Analytics.track(.scriptRunning(script))
```
Comment (vikmeup) [inline Trust/Browser/ViewControllers/BrowserViewController.swift]: Same here

Comment (OlegGordiichuk) [reply Trust/Browser/ViewControllers/BrowserViewController.swift]: done

File: Trust/Browser/Types/DappAction.swift (hunk 4 above)
Comment (OlegGordiichuk) [reply Trust/Browser/Types/DappAction.swift]: done

File: Trust/Browser/Factory/WKWebViewConfiguration.swift [hunk 6]
```diff
@@ -19,10 +19,8 @@ extension WKWebViewConfiguration {
             do {
                 js += try String(contentsOfFile: filepath)
             } catch {
-                NSLog("Failed to load Trust in page provider")
+                Analytics.track(.failedToLoadTrustPageProvider)
```
Comment (vikmeup) [inline Trust/Browser/Factory/WKWebViewConfiguration.swift]: let's remove this.

**Pull request merged**